### ISSUE TITLE
Elevated tau protein in CSF.

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11662]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
The concentration of tau protein in the cerebrospinal fluid (CSF) is above the upper limit of normal.

Comment:
CSF tau protein concentration is typically investigated by immunoassay or mass spectrometry of a CSF sample obtained via lumbar puncture. CSF total tau (t-tau) is considered a biomarker of neuronal degeneration
PMID: 40883301

QUESTION: Why total tau? Does one also measure some other type of tau concentration?

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
